typescript-react/package.json

```diff
@@ -19,7 +19,7 @@
     "@types/chart.js": "^2.9.15",
     "@types/crypto-js": "^3.1.47",
     "@types/jest": "^25.1.4",
-    "@types/moment": "^2.13.0",
+    "@types/gh-pages": "^3.2.2",
```

Comment:
this looks strange?

Reply:
git diff being weird:
- i removed `@types/moment` all together as latest moment version is written in typescript
- `@types/gh-pages` type definitions are added